crypto/conf/conf_mod.c

```diff
@@ -742,7 +743,7 @@ int CONF_parse_list(const char *list_, int sep, int nospc,
     lstart = list_;
     for (;;) {
         if (nospc) {
-            while (*lstart && isspace((unsigned char)*lstart))
+            while (*lstart && ossl_isspace((unsigned char)*lstart))
```

Comment:
This should be kept with isspace(). Config files are supposed to be in the native encoding, i.e. EBCDIC on EBCDIC platforms where ossl_isspace() is always ASCII.